.gitignore

```diff
@@ -14,3 +15,4 @@ tags
 .classpath
 .project
 .settings
+.bloop
```

Comment:
please split commits any refactoring or cosmetic changes 🙏

Reply:
Now it iss in its own commit. thanks for pointing out